### PR TITLE
fix: [PIE-3563]: number formatting and css fix

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
JIRA: https://harness.atlassian.net/browse/PIE-3563

Earlier : 
image

Now: 
Screenshot 2022-05-26 at 5 05 29 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: src/utils/numberFormatter.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+export interface NumberFormatterOptions {
+  truncate?: boolean
+}
+
+export const numberFormatter: (value?: number, options?: NumberFormatterOptions) => string = (
```
Comment (guycastel) [inline src/utils/numberFormatter.ts:12]: I didn't go over the internals of this function but I would check if the built-in library [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/intl/numberformat) is enough to handle what we need.

e.g - this snippet result is `654.32K`:
```
new Intl.NumberFormat(
        'default',
        { notation: "compact" , compactDisplay: "short", unitDisplay: "long", minimumFractionDigits: 0, maximumFractionDigits: 2 }
    ).format(654321);```

Comment (vardanbansal-harness) [inline src/utils/numberFormatter.ts:12]: Might be good idea to have some UTs to verify what this method is doing it actually does it.

**Review by vardanbansal-harness — DISMISSED** [review]

File: src/utils/numberFormatter.ts (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/utils/numberFormatter.ts:12]: ok, will try to do

**Review by guycastel — APPROVED** [review]
Approved with improvement suggestion.

File: src/utils/numberFormatter.ts (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/utils/numberFormatter.ts:12]: sure will check this, thanks for suggesting it.

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**